corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py

```diff
@@ -14,6 +14,7 @@
 
 
 def bootstrap_twilio_gateway(apps, twilio_rates_filename):
+    return
```

Comment:
what is this for? is it a temporary testing thing? (assuming the same applies below)

Reply:
```csv342``` breaks the way I'm using ```csv``` in this file.  However, this management command is obsolete and should get removed anyway.  I'll do that before finalizing this PR.